style.css

```diff
@@ -0,0 +1,241 @@
+/* stylelint-disable no-descending-specificity */
+
+.work {
+  display: grid;
+  grid-template-rows: 1fr;
+  background-color: #e5e5e5;
+  justify-content: center;
+  margin: 0;
+}
+
+.work-title {
+  text-align: center;
+}
+
+.work-title-h1 {
+  font-size: 40px;
+  color: #1c1a19;
+  font-weight: normal;
+  font-style: normal;
+}
+
+.work-title-h1:hover {
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
+
+.work-items {
+  margin-top: 12px;
+  font-family: inherit;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+  padding-left: 12px;
+}
+
+.work-items-button > button {
+  padding-left: 12px;
+  background: #fff;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  color: #091e42;
+  left: 0;
+  top: 0;
+}
+
+.work-items-button > button:hover {
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
+
+.work-items-see > button {
+  width: 124px;
+  height: 48px;
+  background-color: #ff6b00;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: none;
+}
+
+.work-items-see > button:hover {
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
+
+.work-items-project {
+  background-image: url("./img/ImgPlaceholder1.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.work-items-project-button > button {
+  background: rgba(255, 255, 255, 0.24);
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 25px;
+  color: white;
+  border: none;
+  height: 50px;
+  margin-bottom: 22px;
+}
+
+.work-items-project-button > button:hover {
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
+
+.work-items-project > h5 {
+  margin-top: 108px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.work-items-project > p {
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.seeall > button {
+  padding-left: 0;
+  width: 100%;
+  height: 48px;
+  margin-bottom: 0;
+  background: #ff6b00;
+  border: none;
+  color: #fff;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  margin: 24px 0;
+  background-color: #1c1a19;
+  height: 2000px;
+}
```

Comment:
- [ ] Change the font color for this section.
Your page Vs original
<div class="flex">
<img src="https://user-images.githubusercontent.com/65501535/133868322-babf7c27-ca9d-47da-a2be-671927fac1df.png"  style="width: 50%;"/>
<img src="https://user-images.githubusercontent.com/65501535/133868359-ecafe3b4-5a59-494d-9520-b7f52b0cc90a.png"  style="width: 50%;"/>
</div>